pkg/networkgraph/testutils/utils.go

```diff
@@ -18,6 +18,22 @@ import (
 	"github.com/stackrox/rox/pkg/utils"
 )
 
+func AnyFlowWithCluster(toID string, toType storage.NetworkEntityInfo_Type, fromID string, fromType storage.NetworkEntityInfo_Type, clusterId string) *storage.NetworkFlow {
+	return &storage.NetworkFlow{
+		Props: &storage.NetworkFlowProperties{
+			SrcEntity: &storage.NetworkEntityInfo{
+				Type: fromType,
+				Id:   fromID,
+			},
+			DstEntity: &storage.NetworkEntityInfo{
+				Type: toType,
+				Id:   toID,
+			},
+		},
+		ClusterId: clusterId,
```

Comment:
There is currently a function `AnyFlow` that is similar to this, but doesn't have `ClusterId`. I think these flows should always have `ClusterId`. Ideally I would just modify `AnyFlow` but that seems to be outside the scope of this PR.

Reply:
I have now modified `AnyFlow`